lib/workers/repository/config-migration/branch/create.ts

```diff
@@ -30,6 +30,11 @@ export async function createConfigMigrationBranch(
   }
 
   await checkoutBranch(config.defaultBranch!);
+  let contents = migratedConfigData.content;
+  const prettified = await MigratedDataFactory.applyPrettierFormatting();
+  if (prettified) {
```

Comment:
can we throw an error instead of return null, as it should never happen? i like to avoid conditions which are useless at runtime.

Reply:
Instead of throwing, I changed the function signature to accept the needed data, so to not rely on the factory being initialized. but doing so just made it pointless being a static method so I extracted it again to the global scope.
I've also made it a bit more generic.

I also took the opportunity to rename `MigratedData.getAsync()` to `MigratedData.get()`, as I didn't like it that much.